rpc/src/server/router.rs

```diff
@@ -23,6 +23,8 @@ pub(crate) fn create_routes() -> PathTree<Handler> {
     routes.handle("/monitor/heads/:chain_id", handler::head_chain);
     routes.handle("/chains/:chain_id/chain_id", handler::get_chain_id);
     routes.handle("/chains/:chain_id/blocks/:block_id", handler::chains_block_id);
+    routes.handle("/chains/:chain_id/blocks/:block_id/minimal_valid_time", handler::minimal_valid_time);
```

Comment:
@adonagy 
can you please, put/indent all protocol rpcs at the end before "dev rpcs", which will be just routed in the future:
```
// protocol rps
routes.handle("/chains/:chain_id/blocks/:block_id/minimal_valid_time", handler::minimal_valid_time);
routes.handle("/chains/:chain_id/blocks/:block_id/helpers/current_level", handler::current_level);
routes.handle("/chains/:chain_id/blocks/:block_id/helpers/scripts/run_operation", handler::run_operation);
...
```